test/testdrive/timestamps-debezium-kafka.td

```diff
@@ -189,17 +189,6 @@ b  sum
 ------
 1  10
 
-# Under weaker isolation levels we can see the row at time 2 before all objects have closed time 2.
```

Comment:
What happened here?

Reply:
You can no longer view the most recent data in `bar` until `foo` catches up since they are in the same time domain.